src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -360,9 +360,13 @@ def is_ec2_instance_in_account(account_id):
     try:
         # AWS Metadata v2 requires multiple requests in this format as per
         # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html
-        token_api_response = requests.put('http://169.254.169.254/latest/api/token', headers={
-            'X-aws-ec2-metadata-token-ttl-seconds': '21600',
-        })
+        token_api_response = requests.put(
+            'http://169.254.169.254/latest/api/token',
+            timeout=.100,
```

Comment:
is this `0.100`? If so, is it possible to add the `0` to the front, it is a little hard to tell

Reply:
Yup that's right. Changed it to your suggestion in the latest commit